tests/functional/reset_password.js

```diff
@@ -191,19 +191,21 @@ define([
     },
 
     'open complete page with malformed token shows damaged screen': function () {
-      var malformedToken = createRandomHexString(token.length - 1);
       return initiateResetPassword(this, email, 0)
-        .then(openCompleteResetPassword(
-          this, email, malformedToken, code, '#fxa-reset-link-damaged-header'
-        ));
+        .then(function () {
+          var malformedToken = createRandomHexString(token.length - 1);
+          return openCompleteResetPassword(
+            this, email, malformedToken, code, '#fxa-reset-link-damaged-header');
```

Comment:
`this` won't be what you expect inside the function, we'll need to set a reference to the correct this before the `return initiateResetPassword...` and use that instead.